src/Bridge/Symfony/Bundle/Action/SwaggerUiAction.php

```diff
@@ -45,15 +47,17 @@
     private $oauthFlow;
     private $oauthTokenUrl;
     private $oauthAuthorizationUrl;
+    private $oauthRedirectUrl;
     private $oauthScopes;
 
-    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, NormalizerInterface $normalizer, \Twig_Environment $twig, UrlGeneratorInterface $urlGenerator, string $title = '', string $description = '', string $version = '', array $formats = [], $oauthEnabled = false, $oauthClientId = '', $oauthClientSecret = '', $oauthType = '', $oauthFlow = '', $oauthTokenUrl = '', $oauthAuthorizationUrl = '', $oauthScopes = [])
+    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, NormalizerInterface $normalizer, \Twig_Environment $twig, UrlGeneratorInterface $urlGenerator, Packages $assetPackages, string $title = '', string $description = '', string $version = '', array $formats = [], $oauthEnabled = false, $oauthClientId = '', $oauthClientSecret = '', $oauthType = '', $oauthFlow = '', $oauthTokenUrl = '', $oauthAuthorizationUrl = '', $oauthRedirectUrl = NULL, $oauthScopes = [])
```

Comment:
This is a BC break, the new parameter must be the last one, and optional (but it would be better to use the Twig helper, see below)